src/main/java/org/mate/bookstorespringboot/controller/dto/book/BookDto.java

```diff
@@ -1,7 +1,9 @@
 package org.mate.bookstorespringboot.controller.dto.book;
 
 import java.math.BigDecimal;
+import lombok.Builder;
 
+@Builder
```

Comment:
Does it work with record class? https://medium.com/thefreshwrites/migrating-from-lombok-to-records-in-java-fa2ece28bd68